### PR TITLE
feat: enhance moderation experience

### PR DESCRIPTION
**Pull request by geromegrignon**
## Description

related discussion on #446

### Isue Template

`new-framework-support` has been changed from a GitHub Template to a GitHub Form, enforcing the content completion.

The `Describe the solution you'd like ` section is a textarea field supporting markdown format. There is a required validation upon it to be able to submit the issue to create it.

The `Provide a link to a GitHub repo containing an example of the framework` is a simple input field as it only includes an URL. No required validation.

The `Additional context`  is a textarea field supporting markdown format. No required validation.

An example of the experience provided is available [here](https://github.com/kerhub/stackblitz-issue-forms/issues/new?assignees=&labels=&template=new-framework-support.yml&title=Framework+support+for+%5BFRAMEWORK_NAME%5D)

PS: If the required validation doesn't prove itself to be enough to mitigate low quality contributions, I might create a GitHub Action to close related issues opened with the `[FRAMEWORK NAME]` part of the title still present.

### GitHub Action

A GitHub Action has been added to close issues opened with an empty body.
Even if most of these issues are putting little effort and are opened mainly by GitHub new users, I would encourage adding a closing message as a promotion for more meaningful contributions. By automating it with the GitHub Action, it wouldn't require any further work for the maintainers.

An example of the experience provided is available [here](https://github.com/kerhub/stackblitz-issue-forms/issues/new)

### CODE REVIEW

**Comment by geromegrignon** [comment]
Besides the global context, here are some details to review:

- the field type for each section of the GitHub Issue Form (using a simple input field or a textarea)
- the required validation on different fields
- a closing message to be added for the GitHub Action

@jrvidal

File: .github/ISSUE_TEMPLATE/new-framework-support.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: New framework support
+description: Suggest a framework that we should add support for
+title: Framework support for [FRAMEWORK_NAME]
+body:
+  - type: textarea
+    attributes:
+      label: Describe the solution you'd like
+    validations:
+      required: true
+  - type: input
```
Comment (jrvidal) [inline .github/ISSUE_TEMPLATE/new-framework-support.yml:10]: I would make this input mandatory.

File: .github/workflows/close-empty-issues.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+on:
+  issues:
+    types: [opened]
+jobs:
+  closeEmptyIssues:
+    if: "${{ github.event.issue.body == '' }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: close empty issues
+        uses: kerhub/saved-replies@v0.3.0
```
Comment (jrvidal) [inline .github/workflows/close-empty-issues.yml:10]: Let's use the actual hash here:

```suggestion
        // v0.3.0
        uses: kerhub/saved-replies@0123456789abcdef
```

**Review by jrvidal — COMMENTED** [review]
This LGTM @geromegrignon , a few minor comments

**Comment by geromegrignon** [comment]
changes done.

Once merged, I'll create a related PR in the core repository.

**Review by jrvidal — APPROVED** [review]
LGTM :clap:

**Pull request merged**